src/components/PageSections/Menu/_Menu.scss

```diff
@@ -8,39 +8,39 @@
   top: $coa-languageselectbar-height;
   width: 100vw;
   z-index: 2;
-  @include media($coa-large-screen) {
-    background: none;
-    bottom: auto;
-    left: auto;
-    margin: $coa-space-level5 auto 0;
-    overflow-y: visible;
-    position: relative;
-    top: auto;
-    width: 100%;
-    z-index: 2;
-  }
+  // @include media($coa-large-screen) {
```

Comment:
im fine with commenting this out especially for now, but maybe drop a note about why it's commented out and what else is handling the behavior for now

Reply:
The menu button shows/hides when it's smaller than a large screen. This makes it so we stick with the menu button/collapsable menu instead of having the entire nav menu across the top when the screen is big enough